### PR TITLE
FEAT (#138) : update input styles 

### LINKED ISSUE #138: Update input styles to match design

### PR DESCRIPTION
**Pull request by activate-glacier-instinct**
Closes #138

### CODE REVIEW

**Comment by activate-glacier-instinct** [comment]
Hey @geositta 

This we're ready for a review here. 

Tested on all these pages

- Trove
    - [x] ACTIVE
    - [x] ADJUSTING
    - [x] CLOSING
    - [x] OPENING
    - [ ] LIQUIDATED - could not test but implemented the changes blind
    - [ ] REDEEMED - could not test but implemented the changes blind
    - [x] NONE
- Stability
    - [x] NONE
    - [x] DEPOSITING
    - [x] ADJUSTING
    - [x] ACTIVE
- Staking
    - [x] ACTIVE
    - [x] ADJUSTING
    - [x] NONE

File: package.json [hunk 1]
```diff
@@ -57,7 +56,7 @@
     "@typescript-eslint/parser": "^5.18.0",
     "eslint": "^8.12.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-config-next": "^12.1.4",
+    "eslint-config-next": "^12.1.1",
```
Comment (geositta) [inline package.json:59]: was this downgrade for any particular reason?

File: package.json [hunk 2]
```diff
@@ -35,8 +36,6 @@
     "@web3-react/injected-connector": "^6.0.7",
     "ethers": "^5.6.2",
     "framer-motion": "^6.2.8",
-    "@web3-react/walletconnect-connector": "^6.2.13",
```
Comment (geositta) [inline package.json:38]: please add these wallet connectors back!

**Review by geositta — CHANGES_REQUESTED** [review]
There is some code deletion that causes a few breaking changes probably from rebasing. If you could fix those that I have commented on then I can merge this.

My preference would have been to replace the components with the V2 versions rather than keeping the old v1 and creating a separate v2. Was there a technical reason for keeping the v1 other than testing?

File: package.json (hunk 1 above)
Comment (activate-glacier-instinct) [reply package.json:59]: No idea why, something from the rebase most likely. 

git was a bit confused by the incoming changes. 

I'll do a manual comparison to make sure the packages match

File: src/components/NavbarMobile.tsx [hunk 3]
```diff
@@ -37,24 +27,11 @@ export default function NavbarMobile() {
                 alignItems='center'
                 justifyContent='center'
             >
-                <HStack spacing={3}>
```
[the remark this answers is not in the document]
Comment (activate-glacier-instinct) [reply src/components/NavbarMobile.tsx:40]: Another rebasing issue. Will fix it

File: src/components/Staking/NoStake.tsx [hunk 4]
```diff
@@ -9,19 +10,20 @@ export const NoStake: React.FC = () => {
...
-            <Heading>Staking</Heading>
-            <Box sx={{ p: [2, 3] }}>
-                <InfoMessage title={`You have not staked ${GT} yet.`}>
-                    Stake {GT} to earn a share of borrowing and redemption fees.
-                </InfoMessage>
-                <Flex>
+            <HeadingBase>Staking</HeadingBase>
+            <Box>
+                <ActionDescriptionV2>
+                    {`You have not staked ${GT} yet. Stake {GT} to earn a share of borrowing and redemption fees.`}
```
[the remark this answers is not in the document]
Comment (activate-glacier-instinct) [reply src/components/Staking/NoStake.tsx:16]: Nice catch!

**Comment by activate-glacier-instinct** [comment]
```
There is some code deletion that causes a few breaking changes probably from rebasing. If you could fix those that I have commented on then I can merge this.

My preference would have been to replace the components with the V2 versions rather than keeping the old v1 and creating a separate v2. Was there a technical reason for keeping the v1 other than testing?
```

Yeah I'll sort out the rebasing now. 

You're right - I left the V2 components until we tested it. I'll replace the old components for on the next edit.

**Comment by activate-glacier-instinct** [comment]
BTW there was some unused code in the headers on a few pages. 

We had no designs for those items so I just removed the elements, but I'll make a list in case it had a purpose.

File: package.json (hunk 2 above)
Comment (activate-glacier-instinct) [reply package.json:38]: should be there now

**Comment by geositta** [comment]
Thanks for updating! I think my only remaining comment to resolve is adding back the chakra mode function in styles.ts.

**Comment by activate-glacier-instinct** [comment]
Added that now :vulcan_salute:

**Review by geositta — APPROVED** [review]

**Pull request merged**